### DISCUSSION TITLE
Tutorial SSH Git

### DISCUSSION THREAD

**Discussion by psanchezML** [DiffusionPatternsStockMarket/DiffusionPatterns_Repo#6] [Announcements]
Tutoriales de como interactuar con el repo desde la terminal

Windows:
https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository?platform=windows
https://www.youtube.com/watch?v=_HBmV9lT1UE&ab_channel=CodingMSTR

Linux:
https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository?platform=linux

MacOS
https://docs.github.com/en/repositories/working-with-files/managing-files/adding-a-file-to-a-repository?platform=mac
https://www.youtube.com/watch?v=J1BCa0N7Z7c&ab_channel=DigitalCEO

**Comment by Bruno-Luna** [comment]
Very good!
